core/renderers/common/renderer.js

```diff
@@ -13,34 +13,40 @@
 goog.module('Blockly.blockRendering.Renderer');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.blockRendering');
-goog.require('Blockly.blockRendering.ConstantProvider');
-goog.require('Blockly.blockRendering.Drawer');
-goog.require('Blockly.blockRendering.IPathObject');
-goog.require('Blockly.blockRendering.MarkerSvg');
-goog.require('Blockly.blockRendering.PathObject');
-goog.require('Blockly.blockRendering.RenderInfo');
-goog.require('Blockly.Connection');
-goog.require('Blockly.connectionTypes');
-goog.require('Blockly.InsertionMarkerManager');
-goog.require('Blockly.IRegistrable');
-goog.require('Blockly.utils.object');
-
-goog.requireType('Blockly.Block');
-goog.requireType('Blockly.BlockSvg');
-goog.requireType('Blockly.blockRendering.Debug');
-goog.requireType('Blockly.Marker');
-goog.requireType('Blockly.RenderedConnection');
-goog.requireType('Blockly.Theme');
-goog.requireType('Blockly.WorkspaceSvg');
+/* eslint-disable-next-line no-unused-vars */
+const Block = goog.requireType('Blockly.Block');
+/* eslint-disable-next-line no-unused-vars */
+const BlockSvg = goog.requireType('Blockly.BlockSvg');
+const Connection = goog.require('Blockly.Connection');
+const ConstantProvider = goog.require('Blockly.blockRendering.ConstantProvider');
+/* eslint-disable-next-line no-unused-vars */
+const Debug = goog.requireType('Blockly.blockRendering.Debug');
+const Drawer = goog.require('Blockly.blockRendering.Drawer');
+const InsertionMarkerManager = goog.require('Blockly.InsertionMarkerManager');
+const IRegistrable = goog.require('Blockly.IRegistrable');
+const IPathObject = goog.require('Blockly.blockRendering.IPathObject');
```

Comment:
I'm _pretty_ sure `IRegisterable` and `IPathObject` should use `requireType` with suitable `eslint-disable`s.

Reply:
I've updated `IPathObject` and added `eslint-disable`s for both lines, but I will wait on discussion to resolve in  https://github.com/google/blockly/pull/5343 for `IRegisterable`.

Reply:
Updated as part of rebase.